tests/unit/conftest.py

```diff
@@ -673,26 +681,235 @@ def stripe_test_json(request):
 
 
 @pytest.fixture
-def contact_with_stripe_customer(dbsession, example_contact):
+def stripe_customer_data():
+    return {
+        "stripe_id": FAKE_STRIPE_ID["Customer"],
+        "stripe_created": datetime(2021, 10, 25, 15, 34, tzinfo=timezone.utc),
+        # TODO magic string from fxa schema
+        "fxa_id": "6eb6ed6ac3b64259968aa490c6c0b9df",
```

Comment:
_TODO_ here

Reply:
Resolving with this https://github.com/mozilla-it/ctms-api/pull/628#discussion_r1164059629 in mind